packages/flutter/lib/src/services/flutter_version.dart

```diff
@@ -0,0 +1,28 @@
+/// [FlutterVersion] contains various meta information about whith which Flutter build this app was compiled.
+/// This contains the information that is output by executing `flutter --version`.
+/// 
+/// See also:
+///  - [Platform.version](https://api.dart.dev/stable/dart-io/Platform/version.html)
+class FlutterVersion {
+  FlutterVersion._();
```

Comment:
```suggestion
  const FlutterVersion._();
```

Reply:
Does this actually make any difference, since the constructor should never be called anyway?